src/Modal/Modal.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactNode, FC } from 'react'
+import styled, { css } from 'styled-components'
+import { theme } from '../theme'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { Text } from '../Text'
+
+interface IModal {
+  // showModal state
+  showModal: boolean
+}
+
+type ModalProps = {
+  title?: string,
+  body?: string,
+  children?: ReactNode,
+  showModal?: boolean,
+  handleClick: () => void
+}
+
+export const Modal: FC<ModalProps> = ({
+  title = '',
+  body = '',
+  children = '',
+  showModal = false,
+  handleClick,
+}) => {
+
+  return (
```

Comment:
If there's no body to the modal function can we remove the empty line and the return keyword and use the shorthand `}) => ...`